frontend/package.json

```diff
@@ -7,31 +7,39 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "format": "prettier --write './app/**/*.{js,jsx,ts,tsx,scss}'"
+    "format": "prettier --write './app/**/*.{js,jsx,ts,tsx,scss}'",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
   },
   "dependencies": {
-    "axios": "^1.6.8",
-    "better-sqlite3": "^9.6.0",
     "bootstrap": "^5.3.3",
     "bootstrap-icons": "^1.11.3",
-    "drizzle-orm": "^0.30.10",
     "next": "14.2.3",
     "react": "^18",
     "react-bootstrap": "^2.10.2",
     "react-dom": "^18",
     "react-markdown": "^9.0.1"
   },
   "devDependencies": {
-    "@types/better-sqlite3": "^7.6.10",
+    "@chromatic-com/storybook": "^1.6.1",
```

Comment:
Do we really need storybook?

Reply:
We dont strictly need it, but i like the ability to develop and test components in isolation.